components/ecu/ecu_firmware/Core/Src/can.c

```diff
@@ -7,18 +7,25 @@
  */
 
 #include "can.h"
+#include "main.h"
 
 CAN_TxHeaderTypeDef current_message_header = {
-    0x624, 0, CAN_ID_STD, CAN_RTR_DATA, 6, DISABLE
+    0x450, 0, CAN_ID_STD, CAN_RTR_DATA, 3, DISABLE
```

Comment:
Thanks for changing the ID and length; note that the `data` array in `CAN_send_current()` should have the same size as specified here (3 bytes).

Also, please add `#define` statements after the `#includes` for this CAN ID (0x450) and message length (3).